style.css

```diff
@@ -4,6 +4,7 @@
   width: 100%;
   margin: 0%;
   background-color: #e5e5e5;
+  overflow-x: hidden;
 }
 
 .header {
```

Comment:
- [OPTIONAL] As a project requirement you should check that the mobile version is also responsive, which means that it should grow with the screen size until the desktop size:+1:

Reply:
@IngDavidTM on my end everything fits the screen properly maybe can you explain further what needs to be done because I've opened it on 3 different browsers and everything fits perfectly. Your response will be highly appreciated.

Reply:
![Screen Shot 2023-02-21 at 12 23 45 PM](https://user-images.githubusercontent.com/113978708/220303737-dcebc51d-16bd-4337-a791-5f4f4f2de426.png)




@IngDavidTM this is how it looks on my side.

Reply:
@IngDavidTM I don't understand.